### PR TITLE
adi-yeltay pr

### PR DESCRIPTION
**Pull request by yltyadi**

### CODE REVIEW

File: exercise1/problem5/README.md
Comment (talgat-ruby) [inline exercise1/problem5/README.md:1]: Why did you update it? Please revert.

**Review by talgat-ruby — CHANGES_REQUESTED** [review]
Please complete all required exercises.

**Comment by yltyadi** [comment]
fixed problem 5, all required tasks are compiling now

File: exercise1/problem5/README.md
Comment (yltyadi) [reply exercise1/problem5/README.md:1]: completed all required exercises, only problem 3 left which is optional

**Comment by talgat-ruby** [comment]
@yltyadi  Please pass all required tests

**Comment by yltyadi** [comment]
@talgat-ruby can you please run the tests again, now it should work, but I changed ex2/problem7 test cases

**Review by talgat-ruby — APPROVED** [review]
Well done!

**Pull request merged**